src/neuro_relay/__main__.py

```
@@ -233,36 +273,46 @@ async def setup_client_and_link(new_client):
             print("[Main] NakurityLink attached to intermediary")
             # Set up reconnection callback
             client._reconnect_callback = reconnect_client
-    
+
     async def reconnect_client():
```

Ruff (0.14.8)

Missing return type annotation for private function `reconnect_client`

Add return type annotation: `None`

(ANN202)